src/test/java/teammates/test/cases/storage/StudentsDbTest.java

```diff
@@ -33,6 +35,38 @@ public static void setupClass() throws Exception {
     }
     
     @Test
+    public void testDefaultTimestamp() throws InvalidParametersException, EntityAlreadyExistsException, EntityDoesNotExistException {        
+        ______TS("success : created");
```

Comment:
I don't think you need a TS description here. Creating the student attribute is just a preparatory procedure.